### ISSUE TITLE
Mark top bar menu as such for accessibility: 0.5

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#234]
> role = navigation (ARIA), nav (HTML5)
> Will need an aria-label of “Top bar menu” to distinguish it from other navigational elements when out of context.

Traceability: [Microassist Report 1 > Home Page Template > Landmarks](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.7rvkgwq4gpnr)

**Comment by paregorios** [comment]
![image](https://user-images.githubusercontent.com/263285/36857128-588b11e4-1d3d-11e8-8db6-524fd850fb7d.png)

**Comment by paregorios** [comment]
See also [Microassist Report 1 > WCAG Compliance > Menu](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.xtksdz2zjhlm) where we read:

For desktop:
> Given the minimally complex and flat nature of the NYU ISAW site, Microassist does not recommend additional menu widget markup according to WAI-ARIA authoring standards. Wrapping the menu list in a <nav> element with appropriate label should prove sufficient navigational context for assistive technology users. 

For mobile:
> The mobile menu needs to have a separate navigation element that is only implemented during mobile use. When the menu is activated, it needs to follow the same guidelines as the desktop, since there are sometimes hidden submenus.
>
>Mobile menu should contain the following characteristics:
> - Activating icon ideally should be a button.
>   - Links take a user to another page, buttons activate or submit functions
> - Can be activated by touch, mouse click, spacebar or enter.
>   - Links currently do not fully implement all button properties such as spacebarand enter keypress events.
> - Have an accessible label such as an aria-label=”Navigation menu”
> - On open, the menu should put focus on the first focusable item within the menu.
> - User should be forced to stay in the menu until it is closed by activating the close button. The user should not “fall” out of the menu into the main content of the page.
> - Include a close button.

![image](https://user-images.githubusercontent.com/263285/36907529-78241d36-1dfe-11e8-9eab-908ebb13e295.png)

**Comment by paregorios** [comment]
@jessesnyder there's at least 3 tickets specifically about the mobile menu: #266, #267, #268 that I think cover that aspect. The extended quote above is just there to provide context, so I think this ticket is good to go as is without messing with that. Does that seem right to you?

**Comment by jessesnyder** [comment]
@paregorios Ah, yes, that makes sense. One more question: I spent some time researching <nav> and role="navigation", and consulted with an accessibility-obsessed colleague. He recommended *omitting* the role="navigation" in the context of a <nav> element, but I see now it's explicitly recommended in the MicroAssist document. Should I add it?

**Comment by jessesnyder** [comment]
MicroAssist recommends using the "role" attribute consistently throughout their document, so I think we should just stick to that pattern. Updating my branch now.

**Comment by jessesnyder** [comment]
All changes merged to master branch.

**Comment by paregorios** [comment]
Verified fixed and live on production.
